Tests/test_PDBList.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2016 by Jacek Smietanski.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license. Please see the LICENSE file that should have been included
+# as part of this package.
+
+"""Testing access to the PDB over the internet."""
+
+import unittest
+import os
+import shutil
+# We want to test this module:
+from Bio.PDB.PDBList import PDBList
+
+
+class TestPBDListGetList(unittest.TestCase):
+    """
+    Test methods responsible for getting lists of entries.
+    """
+
+    def test_get_recent_changes(self):
+        """
+        Tests the Bio.PDB.PDBList.get_recent_changes method.
+        """
+        pdblist = PDBList()
+        url = pdblist.pdb_server + '/pub/pdb/data/status/latest/added.pdb'
+        entries = pdblist.get_status_list(url)
+        self.assertIsNotNone(entries)
+
+    def test_get_all_entries(self):
+        """
+        Tests the Bio.PDB.PDBList.get_all_entries method.
+        """
+        pdblist = PDBList()
+        entries = pdblist.get_all_entries()
+        # As number of entries constantly grow, test checks if a certain number was exceeded
+        self.assertTrue(len(entries) > 100000)
+
+    def test_get_all_obsolete(self):
+        """
+        Tests the Bio.PDB.PDBList.get_all_obsolete method.
+        """
+        pdblist = PDBList()
+        entries = pdblist.get_all_obsolete()
+        # As number of obsolete entries constantly grow, test checks if a certain number was exceeded
+        self.assertTrue(len(entries) > 3000)
+
+
+class TestPDBListGetStructure(unittest.TestCase):
+    """
+    Test methods responsible for getting structures.
+    """
+
+    def test_retrieve_pdb_file(self):
+        """
+        Tests the process of retrieving files from PDB database.
+        """
+        # This test downloads several large files from external database so it may run a while.
+
+        # Create temporary directory for files downloaded while testing.
+        basedir = os.path.join(os.getcwd(), "pdb-test")
```

Comment:
Using ``tempfile.mkdtemp(...)`` here instead might be better - less chance of leaving temporary files in the source code tree?